go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/bitrise-steplib/steps-restore-carthage-cache
+
+go 1.17
+
+require (
+	github.com/bitrise-io/go-steputils/v2 v2.0.0-alpha.17
+	github.com/bitrise-io/go-utils/v2 v2.0.0-alpha.13
+)
+
+require (
+	github.com/bitrise-io/go-utils v1.0.1 // indirect
```

Comment:
I wonder where the indirect reference to v1 is coming from 🤔

Reply:
I'm glad you asked! `go-steputils/v2` still has a few dependencies on `go-utils/v1`. I'm tracking them here: https://bitrise.atlassian.net/wiki/spaces/~932870153/pages/2452095043/Step+util+library+v2+migration+tracker